tensorboard/plugins/interactive_inference/witwidget/notebook/base.py

```diff
@@ -74,8 +77,11 @@ def __init__(self, config_builder):
       if 'compare_adjust_example' in config else None)
     if 'custom_predict_fn' in copied_config:
       del copied_config['custom_predict_fn']
+      copied_config['uses_custom_distance_fn'] = True
```

Comment:
why is this set here?

Reply:
would you like me to set it where I set custom_predict_fn?
Setting custom_predict_fn should normally be enough, but we have to remove it from copied_config since it cannot be passed to js side so I added this extra key to convey that information.

Reply:
you're setting uses_custom_distance_fn if 'custom_predict_fn' in copied_config, but use_custom_distance_fn is unrelated to if the model has a custom predict function.

Reply:
ups you are correct, thats a good catch. Should test notebook demos without custom_predict_fn as well.